src/Mod/Part/App/Geometry.cpp

```diff
@@ -6203,13 +6203,13 @@ double suggestFilletRadius(const GeomLineSegment *lineSeg1, const GeomLineSegmen
     return r1 < r2 ? r1 : r2;
 }
 
-GeomArcOfCircle *createFilletGeometry(const GeomLineSegment *lineSeg1, const GeomLineSegment *lineSeg2,
+GeomArcOfCircle* create2LinesFilletGeometry(const GeomLineSegment *lineSeg1, const GeomLineSegment *lineSeg2,

```

Comment:
(How) do you plan to extend this to other combinations, like line/arc?

Reply:
Not currently. This PR is only a very basic refactoring without great ambitions. It just aims at encapsulating the arc generation so that it can be used elsewhere.